modules/k8s-nmstate-uninstall-operator.adoc

```diff
@@ -47,6 +48,23 @@ kubernetes-nmstate-operator.v4.18.0   Kubernetes NMState Operator   4.18.0
 $ oc delete --namespace openshift-nmstate clusterserviceversion kubernetes-nmstate-operator.v4.18.0
 ----
 
+. Before you delete the `nmstate` CR, remove the `nmstate-console-plugin` plugin resource from the `nmstate` CR so that `nmstate-console-plugin` does not persist.
```

Comment:
Actually this should be done **after** you deleted the `nmstate` CR.
Because if you do it before, the nmstate-operator is still reconciling that entry in the `consoles.operator.openshift.io` CR and will add it back.